packages/tailwind-config/src/default-config.ts

```diff
@@ -1,23 +1,18 @@
-import semantic from "@telegraph/tokens/mappings/semantic";
-import tokens from "@telegraph/tokens/mappings/tokens";
+import tokens from "@telegraph/tokens/css-variables-map";
```

Comment:
Question for my understanding: so `"@telegraph/tokens/css-variables-map"` now has combined mappings from both `"@telegraph/tokens/mappings/semantic"` and `"@telegraph/tokens/mappings/tokens"`?

Asking because I wasn't sure where that combining happens in this PR, or maybe that was already the case before.

Reply:
Ah, should have called out that I'm removing the semantic tokens in a follow up PR. Just dropped it here to create less work for this PRs focus.

Reply:
Got it! 👍